packages/logger/src/formatters/tests/ConsoleFormatter.test.ts

```diff
@@ -2,6 +2,22 @@ import {ConsoleFormatter} from '../ConsoleFormatter';
 import {LogLevel} from '../..';
 
 describe('ConsoleFormatter', () => {
+  let consoleErrorSpy: jest.SpyInstance;
+  let consoleWarnSpy: jest.SpyInstance;
+  let consoleLogSpy: jest.SpyInstance;
+
+  beforeEach(() => {
+    consoleErrorSpy = jest.spyOn(console, 'error').mockImplementation();
+    consoleWarnSpy = jest.spyOn(console, 'warn').mockImplementation();
+    consoleLogSpy = jest.spyOn(console, 'log').mockImplementation();
+  });
```

Comment:
I wonder if there's a way to do this is one line?

Reply:
Maybe, but I think having an explicit line per function is very easy to read at a glance, I think trying to turn this into a one liner would impact that readability.